src/components/UI/organisms/MenuItem/MenuItem.css

```diff
@@ -0,0 +1,114 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+
+:root {
+  --main: #fad390;
+  --secondary: #636e72;
+}
+
+.MenuItem {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: 1fr repeat(2, 0.5fr) repeat(2, 1fr);
+  grid-column-gap: 0px;
```

Comment:
If you won't pass any value ... so not using this rule, won't it change anything?